fastlane/README.md

```diff
@@ -0,0 +1,40 @@
+fastlane documentation
+----
+
+# Installation
+
+Make sure you have the latest version of the Xcode command line tools installed:
```

Comment:
did this possibly happen when copying the docu from iOS? or do you really need xcode?

Reply:
Funny enough no; this README is automatically generated when a fastlane command is run locally. We could remove the README from the directory. Fastlane is primarily used for Apple platforms but has a decent Android support as well.